diff -urN aalib-1.2.orig/aalib.texinfo aalib-1.2/aalib.texinfo
--- aalib-1.2.orig/aalib.texinfo	Sun Mar  8 21:46:45 1998
+++ aalib-1.2/aalib.texinfo	Mon Apr  5 03:03:36 1999
@@ -1160,7 +1160,7 @@
 It is @code{NULL} by default. Last is pointer to hardware_params structure,
 where driver will put parameters of initialized device.
 This structure holds information about atributes and parameters supported
-by your driver. @xref{Initialization for image saving}
+by your driver. @xref{Initialization for image saving}.
 Just first two fields are used (@code{font} and @code{supported}).
 @var{Font} is pointer to font structure used by hardware. This field should be filed
 at runtime (see @code{aalinux.c} for example), set statically to one of
@@ -1183,7 +1183,7 @@
 will automatically free this pointer at @code{aa_close} if it is non-NULL.
 
 This function should also recomend best available keyboard and mouse drivers.
-@xref{How does the autodetection work}
+@xref{How does the autodetection work}.
 @end deffn
 
 @deffn Function uninit
@@ -1262,7 +1262,7 @@
 key releases too. Note that driver should send releases even @var{mode} is 0.
 
 This function should also recomend best available mouse driver.
-@xref{How does the autodetection work}
+@xref{How does the autodetection work}.
 @end deffn
 
 @deffn Function uninit
